src/common/settings.js

```diff
@@ -751,6 +751,12 @@ const character_settings = {
         "type": "bool",
         "default": false
     },
+    "great-weapon-master-2024": {
+        "title": "Feat: Great Weapon Master 2024 (Apply to next roll only)",
+        "description": "Heavy Weapon Mastery. Apply extra damage equals your Proficiency Bonus.",
+        "type": "bool",
+        "default": false
+    },
```

Comment:
Reading the feat description, I don't think the setting is necessary anymore, since there's no "make a choice before an attack". It seems to always apply, and there's no limit to it, so it should just always add the GWM proficiency bonus on heavy attacks.
https://www.dndbeyond.com/feats/21-great-weapon-master
vs
https://www.dndbeyond.com/feats/1789149-great-weapon-master

The only possibility is if you attack on someone else's turn (opportunity attack), so maybe have a setting but enabled by default? Although the breakdown of the damages is usually meant for people to cherry-pick which damages actually apply depending on the situation, so I wouldn't say it's mandatory either.

Reply:
yep can make those changes np